.eslintrc

```diff
@@ -11,7 +11,8 @@
     "./node_modules/frontend-tools-configs/eslint/flow.js"
   ],
   "rules": {
-    "max-len": ["error", { "code": 120, "ignoreComments": true }]
+    "max-len": ["error", { "code": 120, "ignoreComments": true }],
+    "react/no-multi-comp": "off"
```

Comment:
our configs and eslint versions are so old, that bumping should be performed in a separate PR. in our latest configs i believe we don't have `react/no-multi-comp` rule anymore (or at least allow us to use arrows in JSX, so after bump of deps this could be removed)